.github/workflows/release.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     with:
       publishToBinaries: true
       mavenCentralSync: true
-      slackChannel: team-lang-java-notifs
+      slackChannel: squad-jvm-notifs
```

Comment:
Is this still needed?

Reply:
Yes the gh-action_release does: https://github.com/SonarSource/gh-action_release